### PR TITLE
fixbug: fix Nacos user not found when password has special characters

### PR DESCRIPTION
**Pull request by elrond-g**
### Ⅰ. Describe what this PR did
Nacos's Version 29ba11e has url encoded the password, so the password will be changed when it has special charaters. But at the Version  164ad31, nacos revert the change. So it need to update Nacos Version for fix the bug.

### Ⅱ. Does this pull request fix one issue?

With nacos's issue : https://github.com/alibaba/nacos/issues/3992

### Ⅲ. Why don't you add test cases (unit test/integration test)? 
The case shoud found on Nacos's project.

### Ⅳ. Describe how to verify it
Use password has special charaters, e.g. "elrond%123", "user@12345"

### CODE REVIEW

**Comment by a364176773** [comment]
标题按照规范，英文，bugfix: 开头 补充pr信息及作者到changes文件夹的1.5.0的2个md文件中

**Comment by elrond-g** [comment]
抱歉，已修改

File: changes/1.5.0.md [hunk 1]
```diff
@@ -56,6 +56,7 @@ Seata 是一款开源的分布式事务解决方案，提供高性能和简单
   - [[#3931](https://github.com/seata/seata/pull/3931)] 修复 线程池拒绝执行情况下,dump内存文件名和路径错误的问题
   - [[#3949](https://github.com/seata/seata/pull/3949)] 修复`nacos-config.py`不会跳过空白选项的问题，解决多个分割选项可能导致内容丢失的问题
   - [[#3931](https://github.com/seata/seata/pull/3931)] 修复线程池拒绝执行情况下,dump内存文件名和路径错误的问题
+  - [[#3988](https://github.com/seata/seata/pull/3988)] 修复 nacos 的密码带有特殊字符导致用户名不存在问题
```
Comment (a364176773) [inline changes/1.5.0.md:59]: 这个md里没有登记作者信息

Comment (a364176773) [reply changes/1.5.0.md:59]: 请补充

Comment (elrond-g) [reply changes/1.5.0.md:59]: 尴尬了。。。

**Review by a364176773 — APPROVED** [review]
LGTM

**Pull request merged**